IoTConnect_2.0/IoTConnect_2.0/IoTConnectManager/IoTConnectManager.swift

```diff
@@ -427,10 +471,10 @@ class IoTConnectManager {
                                 // arrData.append([key:[
                                 //                valDictKey:[valDictValue]]])
                                 dictD?.append(anotherDict: [
-                                    "\(key)":[valDictKey:["\(valDictValue)","\(valDictValue)","\(valDictValue)","\(valDictValue)","1.0","\(valDictValue)"]]])
+                                    "\(key)":[valDictKey:["\(valDictValue)","\(valDictValue)","\(valDictValue)","\(valDictValue)","1","\(valDictValue)"]]])
```

Comment:
add "1" to constant

Reply:
@sw-ot-deveshmevada Changes are done in new push